modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/odata/retriever/test/OrganizationODataRetrieverCustomFieldsTest.java

```diff
@@ -253,6 +253,78 @@ public void testGetOrganizationsFilterByCustomFieldWithEqualsAndDateTextType()
 		Assert.assertEquals(organization1, organizations.get(0));
 	}
 
+	@Test
+	public void testGetOrganizationsFilterByCustomFieldWithEqualsAndIntegerKeywordType()
```

Comment:
I'd add a test for `ExpandoColumnConstants.SHORT` and another for `ExpandoColumnConstants.LONG`, does it make sense for you?